crates/nu-command/src/debug/inspect_table.rs

```diff
@@ -48,7 +48,6 @@ pub fn build_table(value: Value, description: String, termsize: usize) -> String
 
     add_padding_to_widths(&mut widths);
 
-    #[allow(clippy::manual_clamp)]
```

Comment:
Ah they made that generally allow again. I think this is only an undefined resolution for float. So the code below could be a safe `clamp`, right?

Reply:
Ah the `.max`/`.min` solution never panics.